src/main/java/net/sf/jabref/gui/EntryMarker.java

```diff
@@ -196,8 +196,6 @@ public static int isMarked(BibEntry be) {
     }
 
     public static boolean shouldMarkEntries() {
-        return (Globals.prefs.getBoolean(JabRefPreferences.MARK_IMPORTED_ENTRIES)
-                && (Globals.prefs.getBoolean(JabRefPreferences.USE_OWNER)
-                        || Globals.prefs.getBoolean(JabRefPreferences.USE_TIME_STAMP)));
+        return Globals.prefs.getBoolean(JabRefPreferences.MARK_IMPORTED_ENTRIES);
```

Comment:
@oscargus: Have a look at `test.java.net.sf.jabref.importer.*`. Thanks to @JabRef/stupro, we have a lot of tests for the importers. Ideally, you should just be able to extend one of the existing importer tests.